### PR TITLE
JavaScript-Best-Practices-Peer-Exercise

### PR DESCRIPTION
**Pull request by Enning94**
## Exercise: JS Best Practices- code review
In this exercise, I:
- created a new repository
- I copied code from Todo List project and committed to a new branch

### CODE REVIEW

File: src/modules/editTask.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { editLocalTodos } from './localStorage.js';
+
+const editTodo = (e) => {
+  const item = e.target;
+  if (item.classList[0] === 'inp') {
```
Comment (ginabeki) [inline src/modules/editTask.js]:
```suggestion
  if (item.classList[0] === 'input') {
```

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      template: './src/index.html',
+    }),
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+    }),
+  ],
+
+  //   optimization: {
+  //     runtimeChunk: 'single',
+  //   },
```
Comment (ginabeki) [inline webpack.config.js]:
```suggestion
```

Comment (ginabeki) [reply webpack.config.js]: Kindly consider removing any comments that don't add anything to your code to comply with KISS and YAGNI principles.

**Review by ginabeki — CHANGES_REQUESTED** [review]

File: src/modules/editTask.js (hunk 1 above)
Comment (ginabeki) [reply src/modules/editTask.js:5]: > 

```suggestion
  if (item.classList[0] === 'inp') {
```
```suggestion
  if (item.classList[0] === 'inp') {
```

Comment (ginabeki) [reply src/modules/editTask.js]: > Kindly consider using descriptive name for your variable to improve readability of your code

Comment (Enning94) [reply src/modules/editTask.js]: Noted, Thank you.

File: webpack.config.js (hunk 2 above)
Comment (Enning94) [reply webpack.config.js]: Thanks for the review 👍

**Pull request merged**